tests/tools/path_history_compression_test.py

```diff
@@ -0,0 +1,27 @@
+import json
```

Comment:
Was this meant to be unit tests for the `path_history_compresstion.py`? Should we move it into the /tests directory?